public/js/togglePassword.js

```diff
@@ -9,3 +9,15 @@ function togglePassword() {
         $('#toggle-icon').removeClass('fa-eye-slash');
     }
 }
+
+function togglePasswordConfirmation() {
```

Comment:
can we use jquery multiple class selectors and use same ```togglePassword()``` function instead of creating new function?

Reply:
I tried that way but  then both the eyes are working together  i.e  if the upper eye is clicked the lower is also toggled or vice versa.

Reply:
You can pass parameters and using conditional statements can counter which element is it of? here you can use 0 and 1.

Reply:
okay , working upon it

Reply:
made the required changes

Reply:
this might not affect much. this just brings code in a single function instead of using two functions and increase cognitive complexity. code blocks for both password fields are still different.

> You can pass parameters and using conditional statements can counter which element is it of? here you can use 0 and 1.

Reply:
so it is fine if we click on one eye button and both the fields toggles together? will it work? @pavanjoshi914

Reply:
no, current implementation using parameters or using two different functions is ok for me!

Reply:
okay I am changing it to two functions and pushing it again please review once .